go/api/server_management_commands_test.go

```diff
@@ -113,3 +113,15 @@ func ExampleGlideClient_InfoWithOptions() {
 
 	// Output: response is of type string
 }
+
+func ExampleGlideClient_LastSave() {
+	var client *GlideClient = getExampleGlideClient() // example helper function
+	response, err := client.LastSave()
+	if err != nil {
+		fmt.Println("Glide example failed with an error: ", err)
+	}
+	fmt.Println(response)
+
+	// Output:
+	// 1742239667
```

Comment:
THis will always fail. Just validate that result > 0 and add a comment to the response.